src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -26,7 +27,7 @@ public static Connection getConnection() {
         try {
             return DriverManager.getConnection(URL, dbProperties);
         } catch (SQLException e) {
-            throw new RuntimeException("Can't create connection to DB ", e);
+            throw new DataProcessingException("Can't create connection to DB ", e);
```

Comment:
it's not a Data Processing exception, because we will throw this exception in case the database connection cannot be established. Let's don't change this line and throw RuntimeException